src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -160,6 +160,15 @@ class TestCaseDisplayService(private val project: Project) {
                         )
                     }
 
+                    // Highlight if line has been deleted
+                    if (event.oldFragment.contains('\n')) {
+                        textFieldEditor.editor!!.markupModel.addLineHighlighter(
+                            DiffColors.DIFF_MODIFIED,
```

Comment:
Wouldn't it make sense to change the color to `Diff_DELETED` instead? Otherwise it just looks like you edited the predecessor-line

Reply:
Indeed, but that would be confusing since you could then modify the line that is colored.

Reply:
Sure, if you insist